rules/windows/process_creation/proc_creation_win_sc_sdset_deny_service_access.yml

```diff
@@ -0,0 +1,41 @@
+title: Deny Service Access Using Security Descriptor Tampering Via Sc.EXE
+id: 99cf1e02-00fb-4c0d-8375-563f978dfd37
+related:
+    - id: 98c5aeef-32d5-492f-b174-64a691896d25 # Generic SD tampering
+      type: similar
+    - id: a537cfc3-4297-4789-92b5-345bfd845ad0 # Specific Technique
+      type: similar
+status: test
+description: Detects suspicious DACL modifications to deny access to a service that affects critical trustee. This can  be used to hide services or make them unstopable.
```

Comment:
```suggestion
description: Detects suspicious DACL modifications to deny access to a service that affects critical trustees. This can be used to hide services or make them unstoppable.
```